### ISSUE TITLE
Container profile incorrectly on most class detail pages

### ISSUE THREAD

**Issue by rmouritzen-splunk** [ocsf/ocsf-server#69]
Most class detail pages are showing the Container profile when the profile is not, in fact, applicable. With the main `ocsf-schema`, 62 out of 64 classes include the container profile with only `base_event` and `incident_finding` not included. The Container profile associated with no classes (per its profile detail page) and is the only profile that is associated with objects.

What might be happening is that class detail "profiles" section is including classes that reference the objects that in turn use the profile, whereas the profile details page is only showing the direct references in the class and object definitions.

**Comment by floydtree** [comment]
Yep, the container profile applies to process object and most of the classes have process object, thus causing the observed behavior.
